### ISSUE TITLE
Error: invalid literal for int()

### ISSUE THREAD

**Issue by BangerTech** [kubawolanin/ha-openhab#2]
After I´ve put the Files in the openhab Folder and restart I get this Error: invalid literal for int() with base 10: ´38.74......´

OH Version 3.2.0.M3
HA Version core-2021.11.1 Home Assistant OS 6.6

<img width="1190" alt="Bildschirmfoto 2021-11-05 um 15 53 14" src="https://user-images.githubusercontent.com/73241309/140530543-c38a88d3-c8ae-4020-a11e-c726f6ac9c4b.png">

<img width="304" alt="Bildschirmfoto 2021-11-05 um 15 42 09" src="https://user-images.githubusercontent.com/73241309/140529914-59d2ec48-ac41-443d-8e09-98ffb15ad6bf.png">

**Comment by kubawolanin** [comment]
It appears that it's a bug in the @sim0nx python-openhab library I use here to communicate with the OH instance:

https://github.com/sim0nx/python-openhab/blob/master/openhab/command_types.py#L230

```

  @classmethod
  def parse(cls, value: str) -> typing.Optional[typing.Tuple[int, int, float]]:
    """Parse a given value."""
    if value in ColorType.UNDEFINED_STATES:
      return None
    hs, ss, bs = re.split(',', value)
    h = int(hs)
    s = int(ss)
    b = float(bs)
    if not ((0 <= h <= 360) and (0 <= s <= 100) and (0 <= b <= 100)):
      raise ValueError()
    return h, s, b
```

it expects `int` whereas your Color item contains float value for `h` and `s`.

**Comment by kubawolanin** [comment]
@BangerTech you could try to identify which item has the problematc value by visiting http://192.168.2.193:8080/rest/items and search for `38.742856` value. 
Then you can try to set its state to be actually integer (round it?) 
Maybe it's some binding that incorrectly sets the value.

`h` is expected to be a big decimal:
https://github.com/openhab/openhab-core/blob/f1319c283a5c5f5bdefffa070ca7d642b71f9b85/bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/HSBType.java#L77

**Comment by BangerTech** [comment]
found it, it's a rgb wifi controller which color state is this number. It's from the Wifi LED Binding. Gotta check for rounding that number. 
Till then thanks a lot buddy!!!!

**Comment by kubawolanin** [comment]
When I'll find time I will submit a PR to python-openhab lib repo to fix this. 👍🏻

**Comment by BangerTech** [comment]
soo I've tried to put a state description over each item but this doesn't work out because the "real" string is still in big decimals and it's coming from the led binding so i guess there is no way on the OH side :-(

**Comment by sim0nx** [comment]
@kubawolanin @BangerTech I fixed that issue in release 2.16.2 ... give a try if you want and let me know if this fixes your issue.
Please feel free to open issues directly at https://github.com/sim0nx/python-openhab :-)

**Comment by BangerTech** [comment]
yeah thanks @sim0nx now it works perfectly fine! thank you very much guys!!
